Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+        data = {"comment": comment}
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/unmute",
+            params=params,
+            json_data=data,
+            headers=headers,
+        )
+
+        return response
+
+    def consult_incidents_updates_through_rest_request(self, offset, items, time):
+        params = assign_params(offset=offset, items=items, time=time, key=self.api_key)
+        headers = self._headers
+
+        response = self._http_request(
+            "GET",
+            "api/incidents/open-incidents/updates",
+            params=params,
+            headers=headers,
+        )
+
+        return response
+
+    def close_incident_request(self, lumu_incident_id, comment):
+        params = assign_params(key=self.api_key)
+        data = {"comment": comment}
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/close",
+            params=params,
+            json_data=data,
+            headers=headers,
+        )
+
+        return response
+
+
+def get_hmac_sha256(key: Optional[str], comment: str):
+    if key is None:
+        key = ''
+    return hmac.new(key.encode(), comment.encode(), hashlib.sha256).hexdigest()
+
+
+def generate_hmac_sha256_msg(key: Optional[str], comment: str):
+    hmac_code = get_hmac_sha256(key, comment)
+    return f"{comment} hmacsha256:{hmac_code}"
+
+
+def validate_hmac_sha256(key: str, comment_w_hmac: str, separator="hmacsha256:"):
+    """
+
+    :param key:
+    :param comment_w_hmac: Resolved, from: ServiceNow Thu Nov 10 2022 07:56:13 GMT-0800 (PST),
+    close_code: Solution provided, close_notes: ddddddddddddd hmacsha256:
+    a9d7947047c371a10a44d07b381152951c46b0c954f8ddf1eeefc29fd120b8ae
+    :param separator:
+    :return:
+    """
+    result = [piece.strip() for piece in comment_w_hmac.split(separator)]
+    if len(result) != 2:
+        return False
+    comment, hash_mac = result
+    # logger.debug(f'{validate_hmac_sha256.__qualname__} comment: {comment}, {hash_mac}')
+    if get_hmac_sha256(key, comment) == hash_mac:
+        # logger.info(f'{validate_hmac_sha256.__qualname__} validating hmac is True: {comment}, {hash_mac}')
+        return True
+    return False
+
+
+def is_msg_from_third_party(key, comment):
+    if comment and validate_hmac_sha256(key=key, comment_w_hmac=comment):
+        return True
+    return False
+
+
+def add_prefix_to_comment(commnet: Optional[str]) -> str:
+    return f"from XSOAR Cortex {datetime.today():%Y%m%d_%H%M%S} {commnet},"
+
+
+def retrieve_labels_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+
+    response = client.retrieve_labels_request(page, items)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveLabels",
+        outputs_key_field="",
```

Comment:
Please update the `outputs_key_field` in all the relevant commands
```suggestion
        outputs_key_field="id",
```

Reply:
added in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7

Reply:
updated in recent commit

Reply:
in some places you set wrong value, for example in line #282:
`outputs_key_field="labels.id"`
the value should be only `id` because the label object primary key is `id` 
please update the CommandResults in your code
see this article for more info:
https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults